### ISSUE TITLE
[Adaptive Card-Schema Explorer]: Captions options is missing in the video which was present under adaptive cards.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7894]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to "Media" link present under "CardElements" from the left navigation using the tab key and activate it using Enter key.
4. Navigate to this video content present under the adaptive card using Tab key and activate it using Enter key.
5. Observe the issue.

Actual Result:
Captions are not provided for the video.

Expected Result:
Captions options should present within the video so that users can turn on it for the captions of the video.

User Impact:
Keyboard users who are with hearing disability will be impacted and will get panic if they cannot see the captions of the video.

MAS Reference Link:
https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.02.02

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/104482612/190641582-e16cccb6-9b38-4e04-9004-26be543aba7a.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
Issue is not repro, hence adding A11yTTValidated tag.
